Clinical-Genomics/scout
# Export dynamic phenotypes with gene symbols to a separate PDF

Fix #2168

Instead of an excel sheet, when clicking on "Download HPO gene panel":
![image](https://user-images.githubusercontent.com/28093618/97853016-d28eb600-1cf7-11eb-8b1d-caad0ba9c46b.png)

- download a PDF file
- with HPO terms as headers
- with ordered genes symbols separated by comma

**How to test**:
1. Install on stage or locally ( if locally run the command `scout --demo update hpo` first)
1. Add more than one HPO terms to a case
1. Generate a dynamic gene list
1. Click on the "Download HPO gene panel"
1. Make sure that it looks OK

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by ML, DN
